pkcs8/src/common.rs

```diff
@@ -0,0 +1,76 @@
+use core::convert::TryFrom;
+
+use der::{Encodable, Encoder, Tagged};
+
+/// Version marker for PKCS#8 documents.
+///
+/// (RFC 5958 designates `0` and `1` as the only valid version for PKCS#8 documents.)
```

Comment:
```suggestion
/// (RFC 5958 designates `0` and `1` as the only valid versions for PKCS#8 documents)
```

(period inside parentheses looks weird)